packages/cf.js/API_REFERENCE.md

```diff
@@ -0,0 +1,50 @@
+# API V0.0.2
+## `cf.js`
+
+- `Client`
+    - Properties
+        - `nodeProvider`
+    - Instance methods
+        - `getApps(): AppInstance[]`
```

Comment:
I'm thinking of ways to support this. The loosest approach would be to group `AppInstances` by their app name or `manifestAddress`. This has a significant security flaw, though: a dapp could query a different dapp's `manifestAddress`, thereby getting the other dapp's `AppInstances`.

Another approach would be to assign an id to an instantiate `AppFactory`, so that when the `AppFactory` installs an `AppInstance`, the `AppInstance` can be assigned the `AppFactory`'s id. The `AppFactory` could then have its own `getApps` method which collects all the `AppInstances` with that `AppFactory` id.

This has implications for the way channels are set up, and I'm curious what @cf19drofxots thinks about that. I'm also not a fan of assigning an id to a factory. It feels like there should be another layer between the factory and the instances.

Reply:
For the sake of demo, we can get away with giving out all the apps for now. Should add a note in the spec about future security measures for authenticating apps and the playground.